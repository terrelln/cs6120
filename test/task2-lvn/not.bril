@main(a: bool): bool {
    b: bool = not a;
    c: bool = not b;
    d: bool = not c;
    e: bool = not d;
    e: bool = not e;
    e: bool = not e;
    e: bool = not e;
    ret e;
}