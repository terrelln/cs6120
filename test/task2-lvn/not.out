@main(a: bool): bool {
.__block0:
  b: bool = not a;
  c: bool = not b;
  d: bool = not c;
  __var4: bool = not d;
  __var5: bool = not __var4;
  __var6: bool = not __var5;
  e: bool = not __var6;
  ret e;
}
